solidity/contracts/token/libs/TokenRouter.sol

```diff
@@ -362,6 +363,25 @@ abstract contract TokenRouter is GasRouter, ITokenBridge {
         uint256 _amountOrId
     ) internal virtual;
 
+    /**
+     * @dev Should transfer `_amount` of tokens from this token router to the fee recipient.
+     * @dev Called by `_calculateFeesAndCharge` when fee recipient is set and feeAmount > 0.
+     * @dev The default implementation delegates to `_transferTo`, which works for most token routers
```

Comment:
Do all instances that override `_transferFee` also have `transferTo` be no-op?

This makes me wonder if the default `_transferTo` function should be default be no-op so that overriders of `_transferFee` don't have to do so.